src/prelude.rs

```diff
@@ -9,9 +9,16 @@ pub use crate::{
         Dispatcher, DispatcherHandlerRx, DispatcherHandlerRxExt, UpdateWithCx,
     },
     error_handlers::{LoggingErrorHandler, OnError},
-    requests::{respond, Request, ResponseResult},
+    respond,
+};
+
+pub use teloxide_core::prelude::*;
+
+pub use teloxide_core::{
+    adaptors::AutoSend,
+    requests::{Request, ResponseResult},
     types::{Message, Update},
-    Bot, RequestError,
+    RequestError,
 };
```

Comment:
Why do we need `ResponseResult`, `Message`, `Update` and `RequestError` here?

Reply:
`Message` and `ResponseResult` are used in our examples extensively (`simple_commands_bot`, for example). `Update` and `RequestError` are superfluous, I think.

Reply:
Hm, right. `Message` is used in handlers quite often, so it's reasonable. Maybe we should add other updates types to prelude too (`InlineQuery`, `CallbackQuery`, etc)?

`ResponseResult`, `RequestError`  - again, tying up the error type doesn't seem right.

`Update` is really redundant.

Reply:
All the other updates types of `UpdateKind`, right?

Reply:
I guess right. The message is probably the most useful one though.